### PR TITLE
Adds VSCode workspace settings + Eloquent JavaScript

### PR DESCRIPTION
**Pull request by remarcmij**
I have replaced the readings from A Smarter Way with readings from Eloquent JavaScript 3rd Edition.

Reason:

- A Smarter Way is based on ES3: no const, let, forEach, map, filter, reduce etc.
- No mention at all of JSON.
- The online PDF version of A Smarter Way looks like a bootleg version.
- Eloquent JavaScript 3rd edition is up to date, covers even async/await. The drawback is that it is a bit harder to read than A Smarter Way.

### CODE REVIEW

File: Week1/MAKEME.md [hunk 1]
```diff
@@ -2,85 +2,52 @@
...
 
 _Deadline Monday_
 
-Your fellow students have provided you with feedback in Trello. 
+Your fellow students and teachers have provided you with feedback on your last JavaScript1 homework in Trello. 
 
 - Implement both feedback from Trello and Github.
 - Check on one of your fellow students code and issues and see if her or she implemented their feedback correctly. If there are some things that can be improved make an issue suggesting further improvements. If you think that the feedback has been implemented correctly create a issue saying something like: "nice work you can clear your issues".
 
-## Step 2: Reorganize your GitHub
```
Comment (mkruijt) [inline Week1/MAKEME.md]: hi Jim why did you move this out?

File: Week1/MAKEME.md [hunk 2]
```diff
@@ -2,85 +2,52 @@
...
+• Code formatting
+• Handing in homework via PR
 ```
 
 >[Here](/Week2/README.md) you find the readings you have to complete before the second lecture.
 
 ## Step 0: Make a small dance
 Give yourself (or your neighbor) a little tap on the shoulder, you've made it to JS2! :muscle:
 
-## Step 0: review
```
Comment (mkruijt) [inline Week1/MAKEME.md]: No review?

File: Week1/README.md [hunk 3]
```diff
@@ -20,16 +22,16 @@ Links to MDN (Mozilla Developer Network) topics:
 
 ## Review
 
-Go through the topics of week 2-4.
+Review through the topics of JavaScript1:
```
Comment (mkruijt) [inline Week1/README.md]: ha it was in there twice

Comment (mkruijt) [reply Week1/README.md]: 👌

File: Week1/README.md [hunk 4]
```diff
@@ -38,15 +40,41 @@ Go through the topics of week 2-4.
...
+- ~~Structuring code~~
 - [Code formatting](./../../../../fundamentals/blob/master/fundamentals/code_formatting.md)
+- [Handing in homework via PR](../../..//fundamentals/blob/master/fundamentals/homework_pr.md)
+
+### Recommended readings
+
+These chapters from _Eloquent JavaScript_ give in-depth explanations of the topics that will be discussed during the lecture. Highly recommended (if time permits).
+
+- Chapter 13: [JavaScript and the Browser](http://eloquentjavascript.net/13_browser.html)
+
```
Comment (mkruijt) [inline Week1/README.md]: great thank you

File: Week1/README.md [hunk 5]
```diff
@@ -38,15 +40,41 @@ Go through the topics of week 2-4.
...
+- Chapter 14: [The Document Object Model](http://eloquentjavascript.net/14_dom.html)
+
+    _You can skip the following sections:_
+
+    - Moving through the tree
+
```
Comment (mkruijt) [inline Week1/README.md]: also thoughtful of you

File: Week1/MAKEME.md (hunk 1 above)
Comment (remarcmij) [reply Week1/MAKEME.md]: Hi Maartje, I moved this out because I think checking the structure should be part of the homework review, at least, that's how I have done it. In the MAKEME there are specific instructions on how to set up the repo and we actually created the `hyf-javascript1` repo on GitHub during the lecture. And of course, as of JS2 we expects student to fork an existing repo.
